_posts/2024/2024-01-15-the-first-five-questions.md

```diff
@@ -0,0 +1,110 @@
+---
+
+I've been talking and writing for several years
+about the need for something that would teach programmers
+the useful basics of economics, politics, sociology, and other social sciences.
+A few days ago I framed it as [the Scratch of social sciences][like-scratch];
+some people interpreted that as, "What software would you use to teach these ideas?"
+but as I later wrote on Mastodon,
+when I say, "X is to the social sciences as Scratch is to computer science,"
+what I'm really looking for is something that combines approachability with a solid foundation
+in the way that Scratch does.
+
+One possibility is something like the [Carpentries][carpentries] workshops.
+A typical workshop spends half a day on the Unix shell,
+half a day on version control,
+and a full day on either Python or R;
+all of the lessons are aimed at people who have little or no previous training in programming
+(though they may often have picked up bits and pieces in a data analysis class).
+
+Could that format be used to teach a few actionable fundamentals of the social sciences?
+I include the word "actionable" because people learn best
+when they can apply their new knowledge immediately to problems they actually face;
+as I've written [elsewhere][t3],
+this means that introductory lessons should focus on "what can I use right away"
+rather than "what's most fundamental".[^1]
+Since I'm expecting to teach programmers[^2],
+I think the best approach would be to teach things related to working in groups on software development projects.
+For example,
+rather than presenting some rules for running meetings,
+a lesson could start by asking,
+"Why do have so many meetings and why do so many of them suck?"
+and use that to motivate discussion of things like
+why large social structures tend to become hierarchical
+and how who's allowed to interrupt whom is tied to perceived social status.
+
+So:
+suppose we have six useful hours in a day of teaching and learning
+(i.e., eight hours minus setup, coffee breaks, lunch, and wrap-up).
+At 45 minutes per episode,
+that gives us eight episodes per day
+or sixteen episodes in total.
+What should the motivating questions for each episode be?
+I've listed four below,
+and I'd like to set up an hour-long call later this month
+for anyone who's interested in expanding on them,
+replacing them with better ones,
+or suggesting another approach entirely.
+If you'd like to take part,
+please [email me](mailto:{{site.author.email}}) and I'll send you an invitation.
+
+Why do we have so many meetings?
+:   Coordination costs rise as the square of the number of people,
+    so large groups must (a)
+    make work uniform
+    so that sets of people are effectively interchangeable (e.g., assembly lines and typing pools),
+    (b) create sub-groups and invest effort in coordinating their work,
+    or (c) allow sub-groups to work independently and pay the cost of duplicated or contradictory effort.
+
+Why do so many meetings suck?
```

Comment:
I think to make this broader meetings topic more actionable, a question could be "How do I make meetings not suck?" and combine the two topics together to address the function of meetings, pitfalls of crappy meetings, and tips for holding a good meeting according to social sciences